lib/navbar/theme.module.scss

```diff
@@ -2,6 +2,7 @@
 
 :local(.navbarWrapper) {
   z-index: 2;
+  display: flex;
   box-sizing: border-box;
   width: 100%;
   height: 81px;
```

Comment:
- height:81px and height: fill-content ??

Reply:
Don't know about this. @KRRISH96 might know. However, I've removed it in this PR. 

@KRRISH96 did it serve any specific purpose?

Reply:
Can Be Removed.
Previously It was fit-content only. 81px is added as a fallback for IE and Edge.